module/module.go

```diff
@@ -269,7 +269,8 @@ func (m *Module) connectParent(ctx context.Context) error {
 	// moduleLoggers may be creating the client connection below, so use a
 	// different logger here to avoid a deadlock where the client connection
 	// tries to recursively connect to the parent.
-	clientLogger := logging.NewDebugLogger("module-connection")
+	clientLogger := logging.NewBlankLogger("module-connection")
```

Comment:
Blank loggers [don't have any appenders/output](https://github.com/viamrobotics/rdk/blob/5ce3ca953af36fcf09ce888bafa9709166d58c98/logging/logging.go#L87). I can be convinced this change being intentional and that we want to swallow logs. But it's definitely not clear why we'd create a blank logger and then change its log level.

Does `client.New` wind up mutating this `clientLogger` in some way that I'm missing?

Reply:
Ah you're right: using a blank logger swallows all logs, and that was not my intention. For context, @cheukt had had an issue where debug logs were getting spewed from the `client.New` call even when the module had a log level that was higher than debug. `client.New` does not mutate the `clientLogger` in any meaningful way from what I can tell. I've switched this to call to be `NewLogger` and left the `SetLevel` below. I don't think we want to swallow all logs; we just want the level here to match that of the module.